db/migrate/20151029212113_add_indices_and_uniqueness.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndicesAndUniqueness < ActiveRecord::Migration
+  def change
+    add_index :users, :email, unique: true
+    add_index :groups, :address, unique: true
+  end
+end
```

Comment:
I don't think we need this for the `users` table.. Again, auth0 will manage users for us. I think we should look at [pc_load_letter](https://github.com/walsh9/pc-load-letter/blob/master/app/models/user.rb) for examples of what we need. 

We probably want to add a uniqueness constraint on the groups `name` attribute as well. (Or do feel like more than two buildings can have the same name? I'm good with either, just think about that decision).